src/osdp_common.c

```diff
@@ -3,8 +3,9 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-
+#ifndef _GNU_SOURCE
 #define _GNU_SOURCE /* See feature_test_macros(7) */
+#endif
```

Comment:
As mentioned in https://github.com/goToMain/c-utils/pull/23, I think we can drop this. Please give it a try and update this PR.